modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/information-banner/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+/* eslint-disable no-undef */
+
+const closeButton = fragmentElement.querySelector('.close');
+const bannerElement = document.querySelectorAll('.cp-information-banner');
+const bannerFragment = fragmentElement.querySelector('.cp-information-banner');
```

Comment:
sort variables where possible. in most cases most items should be sorted alphabetically. when alphabetically is not possible, sort them in a way that initializes them the closest to where they are used.

Reply:
Okay! Thanks.